docs/antrea-ipam.md

```diff
@@ -106,6 +108,15 @@ Antrea deployment YAML are:
     ...
 ```
 
+Alternatively, when if Antrea is installed by helm, you can use the following
+helm installation command to configure the above options:
+
+ ```bash
+ helm install antrea antrea/antrea --namespace kube-system --set
+enableBridgingMode=true,featureGates.AntreaIPAM=true,trafficEncapMode=noEncap,noSNAT=true
```

Comment:
I think you could add "trafficEncapMode=noEncap,noSNAT=true" to the YAML installation too to be consistent.

Reply:
sure. you are so careful!